content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md

```diff
@@ -6,6 +6,27 @@ menu:
     parent: About the project
 ---
 
+
+## v1.5.8 [2020-1-6]
+
+## Breaking changes
+
+- Remove support for darwin/386 builds (Go no longer supports), and add support for darwin/arm64 builds.
+
+## Features
+
+- Add InfluxDB `subscription-path` configuration option to allow Kapacitor to run behind a reverse proxy, thanks @aspring!
+- Add support for HTTP sources in `SideloadNode` configuration, thanks @jregovic!
+- Add support for correlate in the Alerta AlertNode, thanks @nermolaev!
+- Add `details` option to the OpsGenie v2 event handler; set to `true` to use the Kapacitor alert details as OpsGenie description text, thanks @JamesClonk!
+- Preallocate GroupIDs for increased performance by reducing allocations.
+- Send data to InfluxDB compressed as `gzip` by default.
```

Comment:
Do folks have the option to change this in the Kapacitor config and/or elsewhere? If so, do we support other content encoding?

Reply:
They can turn it off by setting compression to "none" in the appropriate influxdb section.

Reply:
Thank you, @docmerlin. Is "the appropriate influxdb section" supposed to be in the [Kapacitor configuration](https://github.com/influxdata/kapacitor/blob/master/etc/kapacitor/kapacitor.conf) or elsewhere? Want to tell folks where they can update this setting.

Reply:
Yes in the kapacitor config were they set how to communicate with influx.

Reply:
@docmerlin, I don't see gzip in the config...should it be there?